gnovm/stdlibs/crypto/ed25519/ed25519.go

```diff
@@ -0,0 +1,9 @@
+package ed25519
+
+import (
+	"crypto/ed25519"
+)
+
+func X_verify(publicKey []byte, message []byte, signature []byte) bool {
```

Comment:
What do you think about renaming this (and in turn the std func) to `VerifyED25519` / `X_VerifyED25519`?

Reply:
Yes, I do it